x/ccv/provider/client/proposal_handler.go

```diff
@@ -148,7 +150,15 @@ type ConsumerAdditionProposalJSON struct {
 	GenesisHash   []byte             `json:"genesis_hash"`
 	BinaryHash    []byte             `json:"binary_hash"`
 	SpawnTime     time.Time          `json:"spawn_time"`
-	Deposit       string             `json:"deposit"`
+
```

Comment:
Leaving this here because Github doesn't let you put comments on stuff that was not changed- 
Can you update the json example starting on line 107?

Reply:
Done.